CustomeDB/LinkedList.cpp

```diff
@@ -216,23 +314,93 @@ void LinkedList::deleteById(int id) {
     }
 }
 
-void LinkedList::displayList() {
+void LinkedList::updateByIdPrj() {
 
-    Node* cur = emphead;
+    if (prjhead == NULL) {
+        cout << "We're sorry but this linked list has same number of entries as you have girls";
+    }
+    else {
+        cout << "Enter the id that you want to update : ";
+        int id;
+        cin >> id;
+        Node* temp = prjhead;
+        bool found = false;
+        while (temp != NULL) {
+            if (temp->prj.getid() == id) {
+                found = true;
+                break;
+            }
+            temp = temp->next;
+        }
+        if (found) {
+            cout << "What do you want to update in this class ? \n";
+            cout << "Press 1 to update name \n";
+            cout << "Press 2 to update description \n";
+            cout << "Press 3 to update client_id \n";
+            cout << "Press 4 to update money earned \n";
+            cout << "Press 5 to update deadline \n";
+            int input;
+            cin >> input;
+
+            if (input == 1) {
+                string name;
+                cout << " enter the new name : ";
+                cin >> name;
+                temp->prj.setname(name);
+            }
+            else if (input == 2) {
+                string description;
+                cout << " enter the new description : ";
+                cin >> description;
+                temp->prj.setdescription(description);
+            }
+            else if (input == 3) {
+                int client_id;
+                cout << " enter the new client_id : ";
+                cin >> client_id;
+                temp->prj.setclient_id(client_id);
+            }
+            else if (input == 4) {
+                string money_earned;
+                cout << " enter the new amount of money earned : ";
+                cin >> money_earned;
+                temp->prj.setmoney_earned(money_earned);
+            }
+            else if (input == 5) {
+                string deadline;
+                cout << " enter the new deadline : ";
+                cin >> deadline;
+                temp->prj.setdeadline(deadline);
+            }
+            else {
+                cout << "Choose betweem given numbers \n";
+            }
+        }
+        if (!found) {
+            cout << "This id is not present in database. \n";
```

Comment:
Why did you add code for input directly inside linkedlist class??? If you want to handle input you can handle it inside Activity class only